includes/blocks/example-block/register.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Example-block
+ * Custom title block -- feel free to delete
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { registerBlockType } from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+import edit from './edit';
+import save from './save';
+import json from './block.json';
+
+const { name } = json;
+
+/**
+ * Register block
+ */
+export default registerBlockType(name, {
+	title: __('Example Block'),
```

Comment:
@ryanwelcher does this not need a text-domain?

Reply:
It does, however my eslint kept removing it on save and I forgot to figure it out 🤦

Reply:
Weird - ignore me.